galaxy/values-cvmfs.yaml

```diff
@@ -213,63 +213,72 @@ useSecretConfigs: false
 
 # All files will be relative to `/galaxy/server/config/` directory
 configs:
-  job_conf.xml: |
-    <job_conf>
-        <plugins>
-            <plugin id="local" type="runner" load="galaxy.jobs.runners.local:LocalJobRunner" workers="4" />
-            <plugin id="k8s" type="runner" load="galaxy.jobs.runners.kubernetes:KubernetesJobRunner">
-              <param id="k8s_use_service_account">true</param>
-              <param id="k8s_persistent_volume_claims">
-              {{- template "galaxy.pvcname" . -}}:{{.Values.persistence.mountPath}},
-              {{- template "galaxy.fullname" . }}-cvmfs-gxy-data-pvc:{{ .Values.cvmfs.data.mountPath }},
-              {{- template "galaxy.fullname" . }}-cvmfs-gxy-main-pvc:{{ .Values.cvmfs.main.mountPath -}}
-              {{- if .Values.extraVolumes }}
-                {{- range $num, $entry := .Values.extraVolumes }}
-                  {{- if $entry.name }}
-                    {{- if $entry.persistentVolumeClaim}}
-                      {{- if $entry.persistentVolumeClaim.claimName }}
-                        {{- range $num, $mount := $.Values.extraVolumeMounts }}
-                          {{- if $mount.name }}
-                            {{- if (eq $entry.name $mount.name) }}
-                              {{- if $mount.mountPath -}}
-                                ,{{- $entry.persistentVolumeClaim.claimName -}}:{{- $mount.mountPath -}}
-                              {{- end }}
-                            {{- end }}
+  job_conf.yml:
+    runners:
+      local:
+        load: galaxy.jobs.runners.local:LocalJobRunner
+        workers: 4
+      k8s:
+        load: galaxy.jobs.runners.kubernetes:KubernetesJobRunner
+        k8s_use_service_account: true
+        k8s_job_api_version: batch/v1
+        k8s_persistent_volume_claims: |-
```

Comment:
Since this is yml, could we change these pvc mappings to a map or list instead of having them as comma separated values?

Reply:
I'm not sure but I'll try it out to see if a list works